### ISSUE TITLE
perPageChange is emitted twice

### ISSUE THREAD

**Issue by mmart1n** [IgniteUI/igniteui-angular#8914]
## Description  
A user is trying to configure an igxGrid with enabled pagination to work with remote data. As stated within our Remote Data Operations topic subscribing to both events onPagingDone and perPageChange is required for correctly fetching the data. I have prepared the following [sample ](https://stackblitz.com/edit/github-per-page-change-emitted-twice)and as you can observe everything is working as expected except for the perPageChange event, which is emitted twice on a single change of the grid's perPage property. This results in sending the request for fetching the data from the Back-End twice.

 * igniteui-angular version: 11.0.x, 10.2.x
 * browser: any

## Steps to reproduce  

1. Open the attached sample
2. Open the console
3. Change items per page

## Result  
perPageChange event is emitted twice.

## Expected result  
perPageChange event should be emitted only once.

**Comment by grosch** [comment]
Any updates on an ETA?

**Comment by hanastasov** [comment]
Hi @grosch, in version 11.1.x, which is about to be released on Monday, 22nd February, this way of doing pagination gets deprecated. Adittional handling that will be needed for remote data scenarios, will need to happen though an external paginator.

https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/remote-data-operations#remote-paging-with-custom-template

This will be updated with some minor fixes on Monday. However, in order not to make you impatient, I will provide sample here around the end of the business day.

Basically everything is the same, just that evertything will happen through the inputs/outputs exposed by the external paginator, rather than through the grid paging properties (which will get deprecated and removed in future version).

**Comment by zdrawku** [comment]
The situation here changed, we had to revert back to the old functionality, so for the time being ignore the last comment by us.
